src/components/common/_material-overrides.scss

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import './components/common/radius';
+
+.mdc-card,
+.mdc-button {
+  border-radius: $BORDER_RADIUS_CARD;
```

Comment:
You should be able to use

@include mdc-button-shape-radius($BORDER_RADIUS_CARD);

and

 @include mdc-card-shape-radius($BORDER_RADIUS_CARD);